tools/syz-reproducers.sh

```diff
@@ -23,7 +23,7 @@ tar -xzf "$target_dir/upstream.tar.gz" -C "$target_dir" && rm "$target_dir/upstr
 mkdir -p "$target_dir/bin"
 
 # Compile the programs and count the successfully built ones.
-built_count=$(find "$target_dir/repros" -name "*.c" -print0 | \
+built_count=$(find "$target_dir/export/bugs" -name "*.c" -print0 | \
```

Comment:
Please add a link to the directory structure somewhere in this file.

Reply:
Added link to the documentation.